config/config_test.go

```diff
@@ -235,7 +235,7 @@ func TestGetConfiguration(t *testing.T) {
 	seedData := config.GetSeedData()
 	assert.Equal(t, 3, len(seedData.Channels))
 	assert.Equal(t, 3, len(seedData.Producers))
-	assert.Equal(t, 3, len(seedData.Consumers))
+	assert.Equal(t, 6, len(seedData.Consumers))
```

Comment:
IMHO, there should be assertions that for old consumers when type is not specified a default is set properly. So we should assert type param is missing and the default is push.

Reply:
Bhaia, I am asserting all possible cases [here](https://github.com/newscred/webhook-broker/pull/140/files#diff-373f5578b8f362a364b50b0d01919a57da64c128547ca10d49df8ae422237c82R293-R310) with consumers' config  
[here](https://github.com/newscred/webhook-broker/pull/140/files#diff-1904722d62de0ec8a93b37b0b06451b0b7c9e66f4eac35e0ce5373bcb379eb2aR86-R104).
Is it good enough?